python_modules/dagster/dagster/_core/storage/cloud_storage_compute_log_manager.py

```diff
@@ -51,6 +51,21 @@ def delete_logs(
     def download_url_for_type(self, log_key: Sequence[str], io_type: ComputeIOType) -> str:
         """Calculates a download url given a log key and compute io type."""
 
+    def uri_or_path_for_type(self, log_key: Sequence[str], io_type: ComputeIOType) -> Optional[str]:
+        """Calculates a download uri given a log key and compute io type."""
+        return None
```

Comment:
maybe worth creating a linear ticket for adding similar handling to other managers